fhir/resources/DSTU2/order.py

```diff
@@ -0,0 +1,164 @@
+        # if property is element of this resource.
+        element_property=True,
+    )
+
+    date: fhirtypes.DateTime = Field(
+        None,
+        alias="date",
+        title="date",
+        description="When the order was made.",
+        # if property is element of this resource.
+        element_property=True,
+    )
+
+    subject: fhirtypes.ReferenceType = Field(
+        None,
+        alias="subject",
+        title="subject",
+        description="Patient this order is about.",
+        # if property is element of this resource.
+        element_property=True,
+        # note: Listed Resource Type(s) should be allowed as Reference.
+        enum_reference_types=["Patient", "Group", "Device", "Substance"],
+    )
+
+    source: fhirtypes.ReferenceType = Field(
+        None,
+        alias="source",
+        title="source",
+        description="Who initiated this order.",
+        # if property is element of this resource.
+        element_property=True,
+        # note: Listed Resource Type(s) should be allowed as Reference.
+        enum_reference_types=["Practitioner", "Organization"],
+    )
+
+    target: fhirtypes.ReferenceType = Field(
+        None,
+        alias="target",
+        title="target",
+        description="Who is intended to fulfill the order.",
+        # if property is element of this resource.
+        element_property=True,
+        # note: Listed Resource Type(s) should be allowed as Reference.
+        enum_reference_types=["Practitioner", "Organization", "Device"],
+    )
+
+    reasonCodeableConcept: fhirtypes.CodeableConceptType = Field(
+        None,
+        alias="reasonCodeableConcept",
+        title="reasonCodeableConcept",
+        description="Why the order was made.",
+        # if property is element of this resource.
+        element_property=True,
+        # Choice of Data Types. i.e reason[x]
+        one_of_many="reason",
+        one_of_many_required=False,
+    )
+
+    reasonReference: fhirtypes.ReferenceType = Field(
+        None,
+        alias="reasonReference",
+        title="reasonReference",
+        description="Why the order was made.",
+        # if property is element of this resource.
+        element_property=True,
+        # Choice of Data Types. i.e reason[x]
+        one_of_many="reason",
+        one_of_many_required=False,
+    )
+
+    when: fhirtypes.OrderWhenType = Field(
+        None,
+        alias="when",
+        title="when",
+        description="When order should be fulfilled. Provide a code or a schedule, but not both.",
+        # if property is element of this resource.
+        element_property=True,
+    )
+
+    detail: ListType[fhirtypes.ReferenceType] = Field(
+        ...,
+        alias="detail",
+        title="detail",
+        description="What action is being ordered.",
+        # if property is element of this resource.
+        element_property=True,
+    )
+
```

Comment:
root validator is missing, please have a look at the example here https://github.com/nazrulworld/fhir.resources/blob/master/fhir/resources/DSTU2/visionprescription.py#L108